docs/src/components/Button/Button.stories.mdx

```diff
@@ -18,54 +18,227 @@ import {Button} from 'dooboo-ui';
+```
 
-#### light
 <Basic themeType="light"/>
-
-#### dark
 <Basic themeType="dark"/>
 
-### Outlined
 
-#### light
-<Outlined themeType="light"/>
+### Solid Color type
+`primary` `secondary` `success` `danger` `warning` `info` `light`
+
+The `type` prop can generate many different buttons.
+The default value of type is `primary`.
+```tsx
+<Button type="primary" text="Primary" style={{padding: 8}} />
+<Button type="secondary" text="Secondary" style={{padding: 8}} />
+<Button type="success" text="Success" style={{padding: 8}} />
+<Button type="danger" text="Danger" style={{padding: 8}} />
+<Button type="warning" text="Warning" style={{padding: 8}} />
+<Button type="info" text="Info" style={{padding: 8}} />
+<Button type="light" text="Light" style={{padding: 8}} />
+```
+
+<Solid themeType="light"/>
+<Solid themeType="dark"/>
+
+
+### Outlined Color type
+Generate bordering button by adding `outlined` prop.
+```tsx
+<Button type="primary" text="Primary" outlined style={{padding: 8}} />
+<Button type="secondary" text="Secondary" outlined style={{padding: 8}} />
+<Button type="success" text="Success" outlined style={{padding: 8}} />
+<Button type="danger" text="Danger" outlined style={{padding: 8}} />
+<Button type="warning" text="Warning" outlined style={{padding: 8}} />
+<Button type="info" text="Info" outlined style={{padding: 8}} />
+<Button type="light" text="Light" outlined style={{padding: 8}} />
+```
 
-#### dark
+<Outlined themeType="light"/>
 <Outlined themeType="dark"/>
 
+
+### Disabled button
+The `disabled` prop can generate disabled button which is not pressed.
+And it can't apply `type`.
+``` tsx
+<Button disabled={true} text="Disabled" style={{padding: 8}} />
+<Button disabled={true} outlined text="Disabled" style={{padding: 8}} />
+```
+
+<Disabled themeType="light"/>
+<Disabled themeType="dark"/>
+
+
+### Loading
+`loading` `indicatorColor`
+
+Set `loading` prop to generate always loading button.
+And change loading spinner color by using  `indicatorColor` prop.
+```tsx
+<Button loading={true} style={{padding: 8}} />
+<Button loading={true} indicatorColor="orange" style={{padding: 8}} />
+```
+
+<Loading themeType="light"/>
+<Loading themeType="dark"/>
+
+
 ### Aspect of sizes 
+`large` `medium` `small`
 
-#### light
-<Sizes themeType="light"/>
+The `size` prop can generate many size buttons. 
+```tsx
+<Button size="large" text="large" style={{padding: 8}}/>
+<Button size="medium" text="large" style={{padding: 8}}/>
+<Button size="small" text="large" style={{padding: 8}}/>
+```
 
-#### dark
+<Sizes themeType="light"/>
 <Sizes themeType="dark"/>
 
-## AdditionalElement
 
-#### light
-<AdditionalElement themeType="light"/>
+### `onPress` event
+When pressing button, some event can generate by using `onPress` prop.
+```tsx
+const [value, setValue] = useState(1)
+  const onIncrease = () => {
+    setValue(value + 1);
+  }
+  const onDecrease = () => {
+    setValue(value - 1);
+  }
```

Comment:
I recommend following [this naming convention](https://jaketrent.com/post/naming-event-handlers-react).

So you can name it `handlePressIncrease`.

Additionally, I personally like to write like this.

```jsx

const handlePress = (offset) => {setValue(value + offset)}

  <Button 
     text='Increase'
     style={{padding: 8}}
     onPress={() => handlePress(1)}
   />
   <Button 
     text='Decrease'
     style={{padding: 8}}
     onPress={() => handlePress(-1)}
   />
```

Why am I doing this? Less event handler is better? 
Maybe. But my point is on `offSet` naming. It makes code self-explainary.

If you want to show intention stronger, you can do something like this. (Although I think It is not needed here.)

```jsx
const handlePress = (offset) => {
  setValue((previous) => previous + offset)
}
```